3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -1,7 +1,13 @@
 import services from '../../../src/app/lib/config/services';
 
 Object.keys(services).forEach(service => {
-  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+  const notWSServices = [
+    'news',
+    'cymrufyw',
+    'naidheachdan',
+    'default',
+    'scotland',
+  ]; // Not WS
 
   if (notWSServices.includes(service)) {
```

Comment:
Shouldn't you be using
```
import config from '../../../support/config/services';
...
```
```suggestion
    if (!config.service.isWorldService) {
```

Reply:
have been told we shouldn't be constantly pulling from that config